### PR TITLE
New visualization tutorial

### PR DESCRIPTION
**Pull request by nathanlct**
## Pull request information

- **Status**: Ready to merge
- **Kind of changes**: Improved tutorial
- **Related PR or issue**: #467 

## Description

I restructured the visualization tutorial (05) and made it nicer. I clearly separated RLlib/rllab/SUMO and made sure there are examples for everything (although I didn't double check the rllab commands are working). 

Next step is to add some pictures to it but I didn't have any at disposal!

### CODE REVIEW

File: tutorials/tutorial05_visualize.ipynb [hunk 1]
```diff
@@ -13,71 +13,92 @@
...
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "collapsed": true
+   },
    "source": [
-    "## Visualization with rllab"
+    "## 1. What is visualization"
```
Comment (Yasharzf) [inline tutorials/tutorial05_visualize.ipynb]: maybe change this to something like "Visualization Components". When you say "What is visualization", I expect to see a definition.

File: tutorials/tutorial05_visualize.ipynb [hunk 2]
```diff
@@ -13,71 +13,92 @@
...
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "collapsed": true
+   },
    "source": [
-    "`frontend.py` requires only one command-line input: the path to the result directory that a user wants to visualize. The directory should contain a `progress.csv` and `params.json` file—pickle files containing per-iteration results are not necessary. An example call to `frontend.py` is below. Click on the link to http://localhost:5000 to view reward over time."
+    "**Contents:**\n",
+    "\n",
+    "[How to visualize using SUMO without training](#2.1---Using-SUMO-without-training)\n",
```
Comment (Yasharzf) [inline tutorials/tutorial05_visualize.ipynb]: it seems the links are broken. Make sure the cell ids are correct

File: tutorials/tutorial05_visualize.ipynb [hunk 3]
```diff
@@ -13,71 +13,92 @@
...
+    "\n",
+    "[How to visualize using SUMO with RLlib](#2.2---Using-SUMO-with-RLlib)\n",
```
Comment (Yasharzf) [inline tutorials/tutorial05_visualize.ipynb]: same

File: tutorials/tutorial05_visualize.ipynb [hunk 4]
```diff
@@ -13,71 +13,92 @@
...
+    "\n",
+    "[How to visualize using SUMO with rllab](#2.3---Using-SUMO-with-rllab)\n",
```
Comment (Yasharzf) [inline tutorials/tutorial05_visualize.ipynb]: same

File: tutorials/tutorial05_visualize.ipynb [hunk 5]
```diff
@@ -13,71 +13,92 @@
...
+    "\n",
+    "[**_Example: visualize data on a ring trained using RLlib_**](#2.4---Example:-Visualize-data-on-a-ring-trained-using-RLlib)\n"
```
Comment (Yasharzf) [inline tutorials/tutorial05_visualize.ipynb]: same

File: tutorials/tutorial05_visualize.ipynb (hunk 1 above)
Comment (nathanlct) [reply tutorials/tutorial05_visualize.ipynb]: Good point, I changed it.

File: tutorials/tutorial05_visualize.ipynb (hunk 2 above)
Comment (nathanlct) [reply tutorials/tutorial05_visualize.ipynb]: They are working for me! (at least now but I didn't change it)

**Comment by AboudyKreidieh** [comment]
@Yasharzf is this ready to be merged?

**Comment by Yasharzf** [comment]
LGTM! Since we have removed rllab, we just need to delete the rllab part.

**Comment by AboudyKreidieh** [comment]
@Yasharzf thanks for the review! I've added the needed changes and will merge now

**Pull request merged**